src/devices/DCMotor/DCMotorWithStartStop.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+using System.Device.Pwm;
+using System.Device.Pwm.Drivers;
+
+namespace Iot.Device.DCMotor
+{
+    /// <summary>
+    /// Direct current (DC) motor with Start/Stop
+    /// </summary>
+    public class DCMotorWithStartStop : DCMotor
+    {
+        private DCMotor _inner;
+        private bool _stopped = false;
+        private double _speed;
+
+        /// <summary>
+        /// Constructs instance with added Start() and Stop() as additional protection
+        /// </summary>
+        /// <param name="innerMotor">Crate DCMotor instance</param>
+        public DCMotorWithStartStop(DCMotor innerMotor)
+            : base(null, false)
+        {
+            _inner = innerMotor;
+            _speed = innerMotor.Speed;
+        }
+
+        /// <summary>
+        /// Enable motor operation.
+        /// </summary>
+        public void Start()
+        {
+            _stopped = false;
+            _inner.Speed = _speed;
+        }
+
+        /// <summary>
+        /// Disable motor operation.
+        /// </summary>
+        public void Stop()
+        {
+            _stopped = true;
+            _inner.Speed = 0.0;
+        }
+
```

Comment:
Maybe you want as well a property called `Enabled` which would be like this:
```csharp
public bool Enabled
{
  get => !_stopped;
  set => value ? Start() : Stop();
}
```

Reply:
I got the error "Only assignment, call, increment, decrement, await, and new object expressions can be used as a statement" so I changed it to:
```csharp
public bool Enabled
{
    get => !_stopped;
    set
    {
        if (value)
        {
            Start();
        }
        else
        {
            Stop();
        }
    }
}
```

Reply:
You're right! Did that without testing.